kapitan/inputs/helm/cffi_build.py

```diff
@@ -5,18 +5,17 @@ def main():
     ffi = FFI()
 
     # declare functions to export
-    ffi.cdef("""
+    ffi.cdef(
+        """
         char* renderChart(char* p0, char* p1, char* p2, char* p3, char* p4, char* p5);
         void free(void* ptr);
-    """)
-
-    ffi.set_source(
-        "helm_binding",  # specify name for importing this module
-        None
+    """
     )
 
+    ffi.set_source("helm_binding", None)  # specify name for importing this module
```

Comment:
interesting to see how black reformatted this block. before, the comment was in the middle of the parameters.